src/vstest.console/Processors/EnableCodeCoverageArgumentProcessor.cs

```diff
@@ -155,11 +155,12 @@ internal class EnableCodeCoverageArgumentExecutor : IArgumentExecutor
 @"            </Functions>" + Environment.NewLine +
 @"            <Attributes>" + Environment.NewLine +
 @"              <Exclude>" + Environment.NewLine +
-@"                <Attribute>^System.Diagnostics.DebuggerHidden.*</Attribute>" + Environment.NewLine +
-@"                <Attribute>^System.Diagnostics.DebuggerNonUserCode.*</Attribute>" + Environment.NewLine +
-@"                <Attribute>^System.Runtime.CompilerServices.CompilerGenerated.*</Attribute>" + Environment.NewLine +
-@"                <Attribute>^System.CodeDom.Compiler.GeneratedCode.*</Attribute>" + Environment.NewLine +
-@"                <Attribute>^System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage.*</Attribute>" + Environment.NewLine +
+@"                <Attribute>^System.Diagnostics.DebuggerHiddenAttribute$</Attribute>" + Environment.NewLine +
```

Comment:
We have this duplicate config here and tracedatacollector. try to keep it in one place.

Reply:
We should check this for 15.8, but I don't want to take this in svc for 15.7